### PR TITLE
Nav Unification: Force classic view on Customizer screens

### PR DESCRIPTION
**Pull request by mmtr**
Fixes https://github.com/Automattic/wp-calypso/issues/55231

#### Changes proposed in this Pull Request:

---

**TL;DR**: This PR forces the classic view (WP Admin) on all the Customizer screens when Nav Unification is enabled.

---

_These changes only affect WP.com sites._

When we removed the global "show wp-admin pages" toggle in https://github.com/Automattic/wp-calypso/pull/54387, we intentionally continued using its value as fallback when a user has still not set a preferred view with the new "Screen Options" tab. See p7DVsv-ch6-p2#comment-36832.

This is problematic on the Customizer and derived screens (such as Widgets or Menus) because we don't render the "Screen Options" tab there due to the different layout, but the global "show wp-admin pages" toggle is still acting on them. That means that users who has active the global toggle, don't have any way now of switching back to the default view.

However, after reviewing all the Customizer screens with @JanaMW27 in both Calypso and WP Admin, we noted that rather than rendering the "Screen Options" tab in these screens, we can actually force the WP Admin views on them, since no functionality is lost (given that all the Calypso views are just iframing the WP Admin views).

#### Jetpack product discussion
N/A.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
- Install Jetpack Beta on an Atomic site and switch to the branch of this PR.
- Go to https://wordpress.com.
- Switch to the Atomic site.
- Make sure that "Appearance > Customizer" opens the WP Admin Customizer.
- Make sure that "Appearance > Widgets" opens the WP Admin Customizer.
- Make sure that "Appearance > Menus" opens the WP Admin Customizer.
- Make sure that "Appearance > Background" opens the WP Admin Customizer.
- Make sure that "Appearance > Additional CSS" opens the WP Admin Customizer.

#### WP.com diffs

In order to unblock the launch of the block editor for widgets on WP.com, the counterpart WP.com changes have been synced manually in different diffs before this PR has been merged:
- D65216-code (deployed in r229917-wpcom).
- D65237-code (deployed in r230015-wpcom).
- D65308-code.

### CODE REVIEW

**Review by retrofox — APPROVED** [review]
LGTM. 👍

**Comment by mmtr** [comment]
Seems that tests failures are unrelated, since they are also failing in `master` right now: https://github.com/Automattic/jetpack/runs/3262093487

Unfortunately, that's blocking the merge of all PRs right now. cc @jeherve not sure if y'all are aware.

**Comment by mmtr** [comment]
I'm merging the WP.com counterpart (D65216-code) in the meantime, so we can unblock the activation of the block editor for widgets on WP.com simple sites.

**Comment by mmtr** [comment]
Deployed to WP.com in r229917-wpcom.

**Comment by mmtr** [comment]
In https://github.com/Automattic/jetpack/pull/20587/commits/cbec15fd9dbd71e86a9ae4632f4f4bf913f08d9d I bringing changes from D65237-code who have been implemented first in WP.com.

**Comment by sdixon194** [comment]
Just confirming that as far as Atomic goes, when you refer to "WP Admin Customizer", we're looking for this screen:

Screen Shot on 2021-08-10 at 11:15:41


And **not** this screen, right? (what would this one be referred to, so I know for future reference?)

Screen Shot on 2021-08-10 at 11:14:57

**Review by sdixon194 — APPROVED** [review]

**Comment by mmtr** [comment]
- D65216-code: deployed in r229917-wpcom.
- D65237-code: deployed in r230015-wpcom.
- D65308-code: deployed in r230139-wpcom.

**Comment by AtrumGeost** [comment]
Hi folks!

I remember this was working at some point last week. But now when I go to `My Sites > Appearance > Widgets` (Simple Site), I land inside the Customizer instead of `wp-admin/widgets.php`. Did anything change?

**Comment by mmtr** [comment]
@AtrumGeost this was requested by Team Calypso in order to unblock the new blocks widgets. See p1628262822091300/1628182071.075900-slack-C7YPUHBB2

**Comment by AtrumGeost** [comment]
Ah, thank you! I can see I completely missed this update:

p1628525963071300/1628262477.060300-slack-CB0B2G43X
